jest.config.js

```diff
@@ -144,20 +144,22 @@ module.exports = {
   // testLocationInResults: false,
 
   // The glob patterns Jest uses to detect test files
-  // testMatch: [
-  //   "**/__tests__/**/*.[jt]s?(x)",
-  //   "**/?(*.)+(spec|test).[tj]s?(x)"
-  // ],
+  testMatch: [
+    "**/__tests__/**/*.[jt]s?(x)",
+    "**/?(*.)+(spec|test).[tj]s?(x)"
+  ],
 
   // An array of regexp pattern strings that are matched against all test paths, matched tests are skipped
-  // testPathIgnorePatterns: [
-  //   "/node_modules/"
-  // ],
+  testPathIgnorePatterns: [
+   "/test/test.js",
+    "/node_modules/"
+  ],
```

Comment:
I am not sure at this point whether or not this `testPathIgnorePatterns` entry is needed, still need to test to be sure.

Reply:
Commenting this out causes the initial test run to fail with Stryker 4.5.1, will keep for now, at least.